setup.py

```diff
@@ -38,4 +39,4 @@ def get_package_version():
     },
     packages=find_packages(include=('zetaforge',)),
     include_package_data=True,
-    package_data={'zetaforge': ['utils/*', 'executables/*'],},)
+    package_data={'zetaforge': ['utils/*', 'executables/*', 'block_maker/*'],},)
```

Comment:
Do those changes belong in this PR?

Reply:
Needs to be merged, I can split it out though.